canonical-web-and-design/ubuntu.com
# Add new certified category views

## Done

Added views for each category
Refactored into a single function that dynamically generates the different category search results

## QA

- Check https://ubuntu-com-10114.demos.haus/certified/desktops
- Check https://ubuntu-com-10114.demos.haus/certified/laptops
- Check https://ubuntu-com-10114.demos.haus/certified/socs
- Check https://ubuntu-com-10114.demos.haus/certified/devices
- Check https://ubuntu-com-10114.demos.haus/certified/servers
Or you can also go to the top navigation and try all the links, they should also work.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4286
Fixes https://github.com/canonical-web-and-design/web-squad/issues/4287
Fixes https://github.com/canonical-web-and-design/web-squad/issues/4288
Fixes https://github.com/canonical-web-and-design/web-squad/issues/4289
